makefiles/boot/riotboot.mk

```diff
@@ -126,6 +126,13 @@ riotboot/slot1: $(SLOT1_RIOT_BIN)
 # Default flashing rule for bootloader + slot 0
 riotboot/flash: riotboot/flash-slot0 riotboot/flash-bootloader
 
+# make applications that use the riotboot feature default to actually using it
+# Target 'all' will generate the combined file directly.
+# It also makes 'flash' and 'flash-only' work without specific command.
+ifneq (,$(filter riotboot, $(FEATURES_USED)))
```

Comment:
This guard is duplicating the one from the file. So is not necessary.

Reply:
yup, fixed